src/main/daml/Daml/Finance/Holding/Base.daml

```diff
@@ -5,7 +5,7 @@ module Daml.Finance.Holding.Base where
 
 import DA.Set (fromList, singleton)
 import Daml.Finance.Interface.Holding.Base qualified as Base (I, View(..))
-import Daml.Finance.Interface.Types.Common.Types (AccountKey(..), InstrumentKey(..), PartiesMap)
+import Daml.Finance.Interface.Types.Common.Types qualified as Types (AccountKey(..), HoldingStandard(..), InstrumentKey(..), PartiesMap)
```

Comment:
is the qualified import required or just for added clarity?

In case you want to use it only for `Types.Base` you can also write `import ... as Types` and then both the unqualified and qualified versions work

Reply:
Base clashes with the holding standard Base

Reply:
I'll change it to
import Daml.Finance.Interface.Types.Common.Types (AccountKey(..), InstrumentKey(..), PartiesMap)
import Daml.Finance.Interface.Types.Common.Types qualified as Types (HoldingStandard(..))